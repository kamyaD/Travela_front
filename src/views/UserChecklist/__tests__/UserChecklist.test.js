import React from 'react';
import { shallow } from 'enzyme';
import { UserChecklist } from '../index';

describe ('UserChecklist Test Suite', () => {
  const props = {
    getOneChecklist: jest.fn(),
    isLoading: false,
    checklist: [{}],
    location: { pathname: '' },
    fullName: '',
  };

  const wrapper = shallow(<UserChecklist {...props} />);
  it('should render correctly', () => {
    expect(wrapper).toMatchSnapshot();
    expect(wrapper.length).toBe(1);
  });

});
