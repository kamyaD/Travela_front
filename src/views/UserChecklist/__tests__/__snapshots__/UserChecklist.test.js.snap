// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserChecklist Test Suite should render correctly 1`] = `
<div
  className="smart-checklist"
>
  <div
    className="smart-checklist-header"
  >
    <div
      className="details"
    >
      <p>
        Your Travel Checklist
      </p>
      Hi  kindly fill in your details to complete your travel checklist
    </div>
    <div
      className="circular-calculator"
    >
      <CircularProgressBar
        percentage={0}
        sqSize={50}
        strokeWidth={5}
      />
    </div>
  </div>
  <div
    className="smart-checklist-body"
  >
    <RenderChecklists
      checklists={
        Array [
          Object {},
        ]
      }
    />
  </div>
</div>
`;
